includes/class-wpcom-legacy-redirector.php

```diff
@@ -64,6 +64,14 @@ static function init() {
 	 * Register custom role using VIP Helpers with fallbacks.
 	 */
 	static function register_redirect_custom_capability() {
+
+		$plugin_version_key = self::POST_TYPE . '_version';
+
+		// Check if this has been run already.
+		if ( self::VERSION <= get_option( $plugin_version_key ) ) {
```

Comment:
The use of the plugin version here means that it will redundantly run on each new plugin version. Instead, we can use a different constant that has an integer just related to the version of these roles and caps changes.

Reply:
I've added a new CAPABILITIES_VER constant for it to fix this with value set to 1.0